include/ucl.h

```diff
@@ -838,6 +842,9 @@ UCL_EXTERN const ucl_object_t* ucl_object_iterate (const ucl_object_t *obj,
  */
 UCL_EXTERN ucl_object_iter_t ucl_object_iterate_new (const ucl_object_t *obj)
 	UCL_WARN_UNUSED_RESULT;
+
+UCL_EXTERN bool ucl_object_iter_chk_excpn(ucl_object_iter_t *it);
```

Comment:
Could you please add doxygen comment for this public function? It's purpose is completely unclear from its name.